x509util/certcheck/certcheck.go

```diff
@@ -35,8 +35,12 @@ var (
 	useSystemRoots            = flag.Bool("system_roots", false, "Use system roots")
 	verbose                   = flag.Bool("verbose", false, "Verbose output")
 	validate                  = flag.Bool("validate", false, "Validate certificate signatures")
-	timecheck                 = flag.Bool("timecheck", false, "Check current validity of certificate")
-	revokecheck               = flag.Bool("check_revocation", false, "Check revocation status of certificate")
+	timeCheck                 = flag.Bool("time_check", false, "Check current validity of certificate")
```

Comment:
Drive-by nit: Would be nice to have all these flags start with `check_` for consistency.

I wish `flag` package had some grouping capability...

Reply:
Done.